guides/customization/tailscale.md

```diff
@@ -3,45 +3,51 @@ title: Tailscale
 description: Learn how to use Tailscale in your Coder workspace.
 ---
 
-This guide describes how to configure [Tailscale networking] for use inside
-[Coder workspaces], so that you can access services running inside Coder as well
-as other services running on your [tailnet (Tailscale private network)]. The
-container image you'll create will include:
+This guide walks you through configuring [Tailscale networking] for use inside
+[Coder workspaces]. With Tailscale networking, you can access services running
+inside Coder and services running on your [tailnet (Tailscale private network)].
 
-- The tailscale daemon (`tailscaled`)
-- A transparent proxy tool, `proxychains4`
+[tailscale networking]: <https://tailscale.com/> [tailnet (tailscale private
+network)]: <https://tailscale.com/kb/1136/tailnet/> [coder workspaces]:
+../../workspaces/index.md
+
+## Creating the image
+
+As part of this tutorial, you'll create an image with the following that you'll
+use to create new Coder workspaces. The container image will include:
+
+- The Tailscale daemon (`tailscaled`)
+- A transparent proxy tool (`proxychains4`)
 - Environment variables to control proxy behavior for outbound connections
-- A [SOCKS5 proxy] listening for connections on `localhost:1080`
-- A [HTTP proxy] listening for connections on `localhost:3128`
+- A [SOCKS5 proxy] to listen for connections on `localhost:1080`
```

Comment:
I think I phrased it as "listening for" because the tailscale daemon is listening for connections, this phrasing seems to imply that there will be a separate proxy listening for connections... do you have suggestions for a clearer/better way to convey what I intended?

Reply:
Dunno if it helps or makes things worse, [tailscale has a diagram](https://tailscale.com/kb/1112/userspace-networking/):

![image](https://user-images.githubusercontent.com/52710/138164373-6d98cb17-a433-4e34-8a02-df6fd66e509e.png)

Does this makes sense to you? Networking is always awful to explain, so I probably did a bad job of it, sorry!

Reply:
Ooo, I like that phrasing. Maybe "An [HTTP proxy] included in tailscaled to facilitate connections to the tailnet, listening on ... "?